File: selection/selection.py
def selectionSort(arr):
	for i in range(len(arr) - 1):
		for j in range(i + 1, len(arr)):
			if arr[j] < arr[i]:
				arr[i], arr[j] = arr[j], arr[i]
	return arr

arr = [7,53,24,45,21,5,78,43,4,25,43]
rt = selectionSort(arr)

print(rt)

File: selection/selection.go
package main
import "fmt"

func main() {
	var arr = []int{7,53,24,45,21,5,78,43,4,25,43}
	selectionSort(arr)
	fmt.Println(arr)
}

func selectionSort(arr []int) []int {
	length := len(arr)
	for i := 0; i < length - 1; i++ {
		min := i
		for j := i + 1; j < length; j++ {
			if arr[min] > arr[j] {
				min = j
			}
		}
		arr[i], arr[min] = arr[min], arr[i]
	}
	return arr
}